### ISSUE TITLE
Region plot 'a' in the legend plot

### ISSUE THREAD

**Issue by karpat90** [totajuliusd/topr#58]
Hello!

Thank you for a great package!

When I am using region_plot(), if the second dataset has no genome-wide significant hits, a red 'a' appears in the plot legend.

Could you help me with this

Thank you

**Comment by totajuliusd** [comment]
Hello,

Can you please show me the command you are using and maybe a screenshot of the plot? I tried to reproduce this using the test data, but everything looks ok when I do:

`regionplot(list(CD_UKBB, UC_UKBB %>% dplyr::filter(P>1e-5)), gene="IL23R")`

Tota

**Comment by karpat90** [comment]
I have used this command

regionplot(list(df2,df), region="chr6:106053916-107309327",
 sign_thresh = 0.01,
             annotate_with_vline = 5e-03,  build = 37,
    legend_labels = c("ph1",              "ph2"),
 protein_coding_only = TRUE, 
![tst](https://github.com/totajuliusd/topr/assets/101727175/afda569a-1d87-4bb5-8b91-93d14e87ded4)
 legend_text_size = 15, axis_text_size = 15, axis_title_size = 15, label_size = 4)

**Comment by totajuliusd** [comment]
Hmmm... that is odd, and something I havent seen before. 

What happens if you do:

```
regionplot(list(df2%>% select(CHROM,POS,P) %>% na.omit(),df %>% select(CHROM,POS,P) %>% na.omit()), region="chr6:106053916-107309327",
         sign_thresh = 0.01,
         annotate_with_vline = 5e-03, build = 37,
         legend_labels = c("ph1", "ph2"),
          protein_coding_only = TRUE,legend_text_size = 15, axis_text_size = 15, axis_title_size = 15, label_size = 4)
```

**Comment by karpat90** [comment]
It's still produces the same figure with 'a'

**Comment by totajuliusd** [comment]
What happens if you replace your dataframes with the test data? 

```
regionplot(list(CD_UKBB,CD_FINNGEN), region="chr6:106053916-107309327",
         sign_thresh = 0.01,
         annotate_with_vline = 5e-03, build = 37,
         legend_labels = c("ph1", "ph2"),
          protein_coding_only = TRUE,legend_text_size = 15, axis_text_size = 15, axis_title_size = 15, label_size = 4)
```

And which version of topr are you using?

**Comment by karpat90** [comment]
![tst2](https://github.com/totajuliusd/topr/assets/101727175/9a8cc9b5-7555-4302-84c9-3957ddd51790)
When I use the command you have suggested I also get plot with NA

**Comment by totajuliusd** [comment]
Which version of topr are you using?  You can use `sessionInfo() `or `?topr `to see the version.

**Comment by karpat90** [comment]
I am using topr_2.0.0

**Comment by totajuliusd** [comment]
Which version of ggplot2 are you using? It looks fine for me and Im using ggplot2_3.4.4. Can you show me the output from the sessionInfo() command?

**Comment by karpat90** [comment]
I am using ggplot2_3.5.0, the latest one

**Comment by totajuliusd** [comment]
That is strange - wich operating system are you on?

**Comment by karpat90** [comment]
I ma using Mac

**Comment by totajuliusd** [comment]
I get the same error after upgrading ggplot2 to version 3.5.0. So this is a
bug in topr that comes with using the most recent version of topr, and
something I need to fix asap.

Until then, you can use topr with an older ggplot version, by doing for
example:

install.packages("
http://cran.r-project.org/src/contrib/Archive/ggplot2/ggplot2_3.4.4.tar.gz",
repos=NULL, type="source")

Thank you for bringing this to my attention!

**Comment by karpat90** [comment]
Unfortunately, when I use ggplot2 version you have suggested, I get 'a' strings elsewhere:
![tst2](https://github.com/totajuliusd/topr/assets/101727175/6e58e0e6-b44e-4fda-b591-5b249063e656)

**Comment by totajuliusd** [comment]
Hi again, 
The code causing the problem has been identified and fixed in topr v2.0.1 which will be submitted to CRAN today, so it should become publicly available within the next few days!

**Comment by totajuliusd** [comment]
topr v2.0.1 is now available on CRAN and downloadable via install.packages
